stylesheet/styles.css

```diff
@@ -0,0 +1,276 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0px;
+  padding: 0px;
+}
+/* Header */
+header {
+  background-color: black;
+  color: white;
+  background-image: url('https://boone824.github.io/adopt-a-dog-winter2020/images/header-image.jpg');
```

Comment:
you should call a relative path to this file `background-image: url('./images.header-image.jpg');`